concrete_datastore/api/v1/views.py

```diff
@@ -2274,6 +2274,17 @@ def handle_divider_update(self, request, request_user, instance):
 
     def update(self, request, *args, **kwargs):
         instance = self.get_object()
+        update_uid = request.data.__contains__('uid')
+
+        if request.method == "PATCH" and update_uid is True:
+            return Response(
+                data={
+                    "message": "The field 'uid' can't be updated",
+                    "_errors": ["INVALID_QUERY"],
+                },
+                status=HTTP_400_BAD_REQUEST,
```

Comment:
You should respond a 400 only if the uid is in the request and it's a different one from the instance.
If the uid didn't change, you should not respond a 400